### ISSUE TITLE
io.deephaven.engine.table.impl.SimpleSourceTable does not render

### ISSUE THREAD

**Issue by devinrsmith** [deephaven/web-client-ui#1763]
As of https://github.com/deephaven/deephaven-core/pull/5078, the following code

```python
from deephaven import parquet

x = parquet.read("my-file.parquet")
x_coalesced = x.coalesce()
```

no longer renders `x` in the UI - it just spins. `x` is of type `io.deephaven.engine.table.impl.SimpleSourceTable`, which is an uncoalesced table (with a single / implicit partition?).

**Comment by mofojed** [comment]
Reproducible using just this snippet:
```python
from deephaven import parquet, new_table
parquet.write(new_table({"Foo": [1, 2, 3]}), "my-table.parquet")
x = parquet.read("my-table.parquet")
```

@rcaudy I want to confirm I'm fixing the right thing. So in the example above, the table `isUncoalesced`, but it has no partition columns. In that case, we're just supposed to fetch the whole table, correct? No partition selector?

**Comment by rcaudy** [comment]
> Reproducible using just this snippet:
> 
> ```python
> from deephaven import parquet, new_table
> parquet.write(new_table({"Foo": [1, 2, 3]}), "my-table.parquet")
> x = parquet.read("my-table.parquet")
> ```
> 
> @rcaudy I want to confirm I'm fixing the right thing. So in the example above, the table `isUncoalesced`, but it has no partition columns. In that case, we're just supposed to fetch the whole table, correct? No partition selector?

Yes, exactly. That's the only way to turn it into something you can render.
